src/components/Zap/BackButton/BackButton.tsx

```diff
@@ -0,0 +1,37 @@
+import ArrowBackIcon from '@mui/icons-material/ArrowBack';
+import { Typography } from '@mui/material';
+import Link from 'next/link';
+import { useTranslation } from 'react-i18next';
+import { BackButtonStyles } from './BackButton.style';
+
+interface BackButtonProps {
+  path?: string;
+  title?: string;
+}
+
+export const BackButton = ({ path, title }: BackButtonProps) => {
```

Comment:
Why not using the back button already existing above?

Reply:
Good question! I did actually try as it seems odd to add a third "BackButton" component. Unfortunately, the layout on the QuestMissionPage and SuperfestMissionPage does not allow this out of the box because of the way how the layout was being applied.
I will add some comments on all BackButton components to return and clean, once we have dedicated time for it.

@tcheee @oktapodia